network/f5/bigip_virtual_server.py

```diff
@@ -508,6 +509,15 @@ def set_default_persistence_profiles(api, name, persistence_profile):
                 profiles=[[{'profile_name': persistence_profile, 'default_profile': True}]]
             )
             updated = True
+        # If a virtual server has a persistence profile and would like this to be None this section enabled this
+        # When persistence_profile is not defined in task or set to Null in vars the persistence profile will be
+        # set to None.
+        if persistence_profile is None or False:
```

Comment:
I could go for that too. It falls back on the module author to interpret emptiness, and not None-ness, as meaning "remove all the profiles"